library/spdm_requester_lib/libspdm_req_get_digests.c

```diff
@@ -184,6 +184,12 @@ libspdm_return_t libspdm_try_get_digest(void *context, uint8_t *slot_mask,
                          spdm_response->digest, digest_size * digest_count);
     }
 
+    spdm_context->connection_info.peer_digest_slot_mask = spdm_response->header.param2;
```

Comment:
The verification of the digests needs to happen after `total_digest_buffer` has been copied.

Reply:
It's fine if it returns an error but wouldn't the integrator want to know the new digest values if verification fails?

Reply:
The problem is: if error is returned, how the caller knows the value is filled or not.

Reply:
But in this case, why you need provision the whole cert chain to trigger the verification process?

Reply:
That's what's in the code now
```
if ((cert_chain_buffer != NULL) && (cert_chain_buffer_size != 0)) {
/* Check if digest of cert_chain_buffer matches a digest in the DIGEST response. */
```

Reply:
> That's what's in the code now

I am sorry, what is feedback ?